vfox-restServer/src/main/java/com/vfoexchange/restServer/controllerAdvice/CustomizedResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.vfoexchange.restServer.controllerAdvice;
+
```

Comment:
Remove unnecessary blank line.

Reply:
Unnecessary blank line removed and committed.